tools/serverpod_cli/bin/generator/types.dart

```diff
@@ -154,7 +154,7 @@ class TypeDefinition {
     if (className == 'double') return 'double precision';
     if (className == 'DateTime') return 'timestamp without time zone';
     if (className == 'ByteData') return 'bytea';
-
+    if (className == 'Duration') return 'integer';
```

Comment:
Actually, in Postgres this needs to be a `bigint` as `integer` only holds a 32 bit value (which may not be enough for the Duration, Dart uses 64 bit integers by default).

Reply:
Yes sure